less/list.less

```diff
@@ -26,11 +38,19 @@
   &__image-container {
     align-items: flex-start;
     justify-content: center;
-    width: 20%;
+    width: 80px;
```

Comment:
Regarding setting a width on the icons:

- I like the idea of being able to keep the images a consistent size.  Using a width in the less achieves this.
- That said, maybe this width could be a variable that can be easily changed via the theme?  @list-icon-width or something.
- Without setting a width, SVGs that do not include a set width/height could collapse to 0x0.

Reply:
I agree to swapping px for rem. Using a width variable would offer flexibility but I'm not sure what's best practice with non-core plugins as this variable wouldn't exist in Vanilla.

Reply:
I'm not keen on the width fixed as it forces that size and its then unchangeable which seems odd to me. Do we do this anywhere else? I'm not sure about a variable/config for the width seems unnecessary but perhaps it could just come down to how much config options AAT is going to need going forward.

Reply:
We don't do this elsewhere but this is a bit of a niche case as the list layout (not column) assumes a typical list format - an icon style image rather than a content graphic. I think without the width property, even when images are the same dimension the image/text width can vary across items depending on text length. With the prior 20% width, I personally found this too big and was having to manually change it on every project. 
I think configurable widths could be overkill but if the need arises this could be picked up at a later date. I think 5rem is a good default and will suit most use cases. There's always the option of overriding this in the theme if needed too.

Reply:
This is largely due to the css, if done a different way then the width would not need to be set at all. In my experience I'm always setting the width or changing the css.